lib/airbrake/sidekiq/retryable_jobs_filter.rb

```diff
@@ -13,6 +13,10 @@ class RetryableJobsFilter
         DEFAULT_MAX_RETRY_ATTEMPTS = ::Sidekiq::JobRetry::DEFAULT_MAX_RETRY_ATTEMPTS
       end
 
+      def initialize(max_retries: nil)
```

Comment:
Perhaps, it would be better not to shadow the method call to avoid headaches with precedence. I think this might work:

```rb
def initialize(max_retries: nil)
  @max_retries = max_retries || find_max_retries
end
```

Where `find_max_retries` is the current `max_retries` method. This will fix the condition on line 36.

Reply:
Mmm the problem is that the precedence must go: `@max_retries` (the option) => `job['retry']` => `max_retries` (the method). I don't think we can combine the `@max_retries` option with the value returned by `max_retries` (or `find_max_retries`) and still retain that order of precedence.